index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Capstone Project Module 1 - Home Page</title>
+
+  <!-- My Styles -->
+  <link rel="stylesheet" type="text/css" href="css/styles.css" />
+
+  <!-- Font Awesome Icons -->
+  <script src="https://kit.fontawesome.com/64d25ab310.js" crossorigin="anonymous"></script>
+
+  <!-- Google Fonts -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Ubuntu:wght@300;400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header id="desktop-header">
+    <nav class="desktop-nav-list display-flex">
+      <a class="mobile-menu-a main-logo" href="index.html"><img src="images/tchaikovski-img.jpg" alt="An image of Tchaikovski"></a>
+      <ul class="desktop-menu-links display-flex">
+        <li><a class="desktop-menu-a" href="about-page.html" id="about-link-d">About</a></li>
+        <li><a class="desktop-menu-a" href="#program-section">Program</a></li>
+        <li><a class="desktop-menu-a" href="#repertoire">Repertoire</a></li>
+        <li><a class="desktop-menu-a" href="#sponsors">Sponsors</a></li>
+        <li><a class="desktop-menu-a" href="#">News</a></li>
+        </ul>
+    </nav>
+  </header>
+
+  <div class="first-container">
+    <header id="mobile-header">
+      <nav class="nav-list">
+        <div id="nav-bar-btn">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+
+        <div class="mobile-menu">
+          <ul class="mobile-menu-links display-flex">
+            <li><a class="mobile-menu-a main-logo" href="index.html"><img src="images/tchaikovski-img.jpg"  alt="An image of Tchaikovski"></a></li>
+            <li><a class="mobile-menu-a" href="about-page.html" id="about-link-m">About</a></li>
+            <li><a class="mobile-menu-a" href="#program-section">Program</a></li>
+            <li><a class="mobile-menu-a" href="#repertoire">Repertoire</a></li>
+            <li><a class="mobile-menu-a" href="#sponsors">Sponsors</a></li>
+            <li><a class="mobile-menu-a" href="#">News</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
```

Comment:
- [ ] Kindly add the dark top bar containing social media links, logout, and language links aligned to the right on top of the menus
![Screen Shot 2022-05-20 at 07 49 56](https://user-images.githubusercontent.com/28307960/169459494-3c21e898-a367-4b59-984c-d9cedf92f866.png)

Your design
![Screen Shot 2022-05-20 at 07 50 18](https://user-images.githubusercontent.com/28307960/169459560-145dec4b-5613-4b6e-90d3-611f8876c26a.png)

Reply:
Addressed ✔